doc/tutorial/modes.txt

```diff
@@ -197,6 +197,33 @@ DebugMode      no         yes                VERY HIGH  Make many checks on what
 
 For more detail, see :ref:`Mode<libdoc_compile_mode>` in the library.
 
+.. _optimizers:
+
+Optimizers
+==========
+
+Theano allows compilations with a number of predefined optimizer modes.
+An optimizer mode consists of a particular set of optimizations, that speed
+up execution of Theano programs.
+
+The optimizers Theano provides are summarized below to indicate the trade-offs
+one might make between compilation time and execution time.
+
```

Comment:
Add something like:

They can be enabled globally with the Theano flag: optimizer=name
or per call to theano.function with theano.function(...mode=theano.Mode(optimizer="name"))